src/chains/ethereum/ethereum/src/wallet.ts

```diff
@@ -411,7 +413,7 @@ export default class Wallet {
     }
 
     if (!localMac || !mac.toBuffer().equals(localMac)) {
-      throw new Error("could not decrypt key with given password");
+      throw new Error("could not decrypt key with given passphrase");
```

Comment:
Just an observation, this error message seems similar to another one later in the file that includes `(default passphrase for accounts created at startup is "")`.

Reply:
Yes, this is intentional! The one stating `(default passphrase for accounts created at startup is "")` is just extra specific because we know in that case that they didn't provide a passphrase at startup, so they need to send `""` to decrypt.